packages/ava/src/utils/dataFormat.ts

```diff
@@ -21,5 +21,5 @@ export default function dataFormat(value: number | string, digits: number = 2) {
     .find((item) => {
       return value >= item.value;
     });
-  return item ? (value / item.value).toFixed(digits).replace(rx, '$1') + item.symbol : '0';
+  return item ? (value / item.value).toFixed(digits).replace(rx, '$1') + item.symbol : value.toFixed(digits);
```

Comment:
What is the usage scenario? Should we add more comments for the func?

![image](https://user-images.githubusercontent.com/15177000/227205091-5cac04d1-fe0e-4296-a0d8-8f66d7a7293a.png)

Reply:
there is a bug in original dataFormat function. If value <1 , item will be undefined and will return 0, due to following code:
<img width="510" alt="截屏2023-03-24 下午3 23 23" src="https://user-images.githubusercontent.com/16997933/227452639-cde91bcf-b0bd-4e9c-a4c3-4308dff7d139.png">

but a value  less than 1 should be formatted as 0.xx by default, otherwise the correlation coefficient will be displayed as 0

Reply:
got it